Python/Product/PythonTools/PythonTools.vsct

```diff
@@ -285,8 +285,8 @@ permissions and limitations under the License.
         <Strings>
           <ButtonText>Send to &amp;Interactive</ButtonText>
           <ToolTipText>Sends the current editor selection to a Python Interactive window.</ToolTipText>
-          <CanonicalName>Send Selection To Interactive</CanonicalName>
-          <LocCanonicalName>Send Selection To Interactive</LocCanonicalName>
+          <CanonicalName>.Python.Send Selection To Interactive</CanonicalName>
+          <LocCanonicalName>.Python.Send Selection To Interactive</LocCanonicalName>
```

Comment:
Don't add the prefix on the localized name (and if you want, remove it from the one localized name it is already on - the rest look fine)